hydra_python_core/doc_maker.py

```diff
@@ -105,9 +105,11 @@ def create_doc(doc: Dict[str, Any], HYDRUS_SERVER_URL: str = None,
     if HYDRUS_SERVER_URL is not None and API_NAME is not None:
         apidoc = HydraDoc(
             API_NAME, _title, _description, API_NAME, HYDRUS_SERVER_URL, doc_name)
-    else:
+    elif entrypoint.get('@id'):
```

Comment:
this condition is ambiguous as the method returns `None`. A better and more readable solution is `entrypoint.get('@id') is not None`